Proposed Documentation Improvements (#2883)
yewstack/yew: website/versioned_docs/version-0.19.0/concepts/wasm-bindgen/introduction.mdx

can be used again if the cast failed and thus returned `None`. The
[`dyn_into`](https://rustwasm.github.io/wasm-bindgen/api/wasm_bindgen/trait.JsCast.html#method.dyn_into)
method will consume `self`, as per convention for into methods in Rust, and the type returned is

> `Result<T, Self>`. This means if the casting fails then the `Self` value in `Err` preserved. So you can try again

Reviewer: ```suggestion
`Result<T, Self>`. If the casting fails, the original `Self` value is returned in `Err`. You can try again
```
